test/attachment-reference.spec.js

```diff
@@ -14,7 +14,7 @@ describe('Attachment reference validation type', () => {
       it('allows an attachment reference with a valid file extension', () => {
         const doc = {
           _id: 'foo',
-          type: 'attachmentsDoc',
+          type: 'attachmentReferencesDoc',
```

Comment:
I made the document type's name more precise so it's easier to identify which file has the problem if there are any joi validation errors.